shell/server/stats-server.js

```diff
@@ -182,6 +182,10 @@ function recordStats() {
     plans: planStats,
   };
   record.computeTime = Date.now() - now;
+  if (SandstormDb.getTotalCharges) {
+    // This only exists under Blackrock
+    record.totalCharges = SandstormDb.getTotalCharges();
```

Comment:
BlackrockPayments is weird in that it may or may not exist, and checking for its existence is annoying (you have to check in globals). Attaching Blackrock methods to SandstormDb is the pattern we've used elsewhere.